src/vni-manager.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * File:  vni-manager.js
+ */
+var _ = require('underscore');
+
+var inputStates = require('./input-states');
+
+var IIP_VNID = '';
+
+/**
+ * Manage the node's vni's by vnid.  If no vnid is provided the IIP VNID will be used.
+ * If the vni for the specified vnid does not exist, a new one will be created.
+ *
+ * @this the component node instance unless otherwise specified on an API
```

Comment:
General comment: There are several places in the code where I found the names confusing.  I think the naming throughout this file should be aligned with the naming conventions that we decided last week:
https://docs.google.com/document/d/1mb_8DCV7nnvhYwc0Ns_gCWpzyXcM1_8qs2u9GJpIZP4/edit#

- An instance of a NoFlo component will be called a "node" (not nodeInstance);
- A NoFlo component (actually a node factory) will be called a "component".  
 - I don't know if we decided what to call the component facade, but how about "componentFacade"?